### ISSUE TITLE
[Feature] Rule to suggest sprintf instead of String.Format

### ISSUE THREAD

**Issue by knocte** [fsprojects/FSharpLint#542]
### Description

Rule to suggest sprintf instead of String.Format

### Testcases

Code that should be flagged:

1. 
```
let someString = String.Format("Hello {0}", world)
```

Code that should not be flagged:

1. 
```
let someString = sprintf "Hello %s" world
```

2. 
```
let someString = String.Format(someTemplate, world)
```
